apps/spruce/src/components/Header/Navbar.tsx

```diff
@@ -72,7 +72,7 @@ export const Navbar: React.FC = () => {
           onClick={() => sendEvent({ name: "Clicked logo link" })}
           to={routes.myPatches}
         >
-          <AnimatedIcon icon={FallTree} />
+          <AnimatedIcon icon={HolidayTree} />
         </LogoLink>
```

Comment:
super nitpick but I think the icon is slightly uncentered (vertically, not horizontally) and could be moved up by about 4px